### ISSUE TITLE
it does not work on IE11+Win10

### ISSUE THREAD

**Issue by o-jill** [o-jill/washcrus#79]
on IE11+Win10, pieces does not appear on the board and TEGOMAs and resign button are appeared.
so initialization does not work.

**Comment by o-jill** [comment]
IE does not support a default value for an argument.
ref: https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Functions_and_function_scope/Default_parameters

```javascript
/*
 * this does not work on IE
 */
function func1(arg = 'default value') {
 do_something(arg);
}

/*
 * this may work on IE
 *
 * but you can not put zero as arg.
 */
function func1(arg) {
 arg = arg || 'default_value'
 do_something(arg);
}
```

**Comment by o-jill** [comment]
only 1 function uses default value in argument.

```javascript
// ui.js@line1604
/**
 * @param {String} nth 何手目
 */
function gensfen(nth = '1') {
  // do_something
}


// ui.js@line1895
function record_your_move()
{
 taikyokuchu = false;

 nteme = mykifu.NTeme;
 // nteme = document.getElementById('nthmove').innerHTML;
 gensfen(nteme);

 send_csamove();
}
```

**Comment by o-jill** [comment]
fixed in b772f23a8e0.
